events/messageFeatures/messageFilter.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * MessageFilter
+ *  For now, this only applies to the MainGuild or TestGuild
+ *
+ * @func messageFilter
+ *  Deletes/filters messages
+ *  Returns whether or not a message has been deleted/filtered
+ *  This will decide what to do with message for message filters
+ * @param {*} message The message in question
+ * @param {*} settings Settings for the bot
+ * @returns {Boolean} allowMessage?
```

Comment:
Should be
```suggestion
 * @returns {Boolean} deleteMessage?
```
the value it returns determines whether to delete the incoming message or not (`true` => delete, `false` => DON'T delete)